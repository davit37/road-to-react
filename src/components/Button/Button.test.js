import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import {Button} from './index';


Enzyme.configure({adapter: new Adapter()})

describe('Button',()=>{
    it('Test Button ', ()=>{

        const el = shallow(
            <Button>Hide</Button>
        )

        expect(el).toMatchSnapshot();
    })
})