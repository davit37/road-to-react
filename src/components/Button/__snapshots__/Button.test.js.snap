// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Test Button  1`] = `ShallowWrapper {}`;
